lsp/astro.lua

```diff
@@ -1,6 +1,6 @@
 local function get_typescript_server_path(root_dir)
   local project_root = vim.fs.dirname(vim.fs.find('node_modules', { path = root_dir, upward = true })[1])
-  return project_root and (project_root .. '/node_modules/typescript/lib') or ''
+  return project_root and vim.fs.joinpath(project_root, 'node_modules', 'typescript', 'lib') or ''
```

Comment:
string literals don't need to be joined. just use "/".

     vim.fs.joinpath(project_root, 'node_modules/typescript/lib')

Reply:
right it gsub path separator according to the system.

Reply:
"/" works everywhere. platform-specific slash isn't needed.